python/grass/jupyter/interactivemap.py

```diff
@@ -347,13 +348,116 @@ def add_layer_control(self, **kwargs):
         else:
             self.layer_control_object = self._ipyleaflet.LayersControl(**kwargs)
 
+    def draw_computational_region(self):
+        # import ipywidgets as widgets
+        # import pyproj
+        # from shapely.geometry import box
+        # from shapely.ops import transform
+
+        # region_mode_button = widgets.ToggleButton(
+        #     icon="square",
+        #     value=False,
+        #     tooltip="Click to add computational region",
+        #     button_style="info",
+        #     layout=widgets.Layout(width="33px", margin="0px 0px 0px 0px"),
+        # )
+
+        # bottom_output_widget = widgets.Output(
+        #     layout={
+        #         "width": "100%",
+        #         "max_height": "300px",
+        #         "max_width": "300px",
+        #         "overflow": "auto",
+        #     }
+        # )
+
+        # region_coordinates = {}
+        rectangle = None
+        latlon_bounds = get_computational_region_bb()
+        print(latlon_bounds)
+        latlon_bounds = get_computational_region_bb()
```

Comment:
Check the order of coordinates - it probably should be latitude, longitude.

Reply:
Umm, I don't think so since get_computational_region_bb() returns [(s, w), (n, e)], which is the exact format required by `ipyleaflet.rectangle`.

Reply:
Your function returns it in the wrong order.